api/capi/include/nntrainer.h

```diff
@@ -215,6 +215,43 @@ int ml_train_model_set_optimizer(ml_train_model_h model,
 int ml_train_model_set_dataset(ml_train_model_h model,
                                ml_train_dataset_h dataset);
 
+/**
+ * @brief Save the model after training
+ * @details Use this function to save the current model
+ *
+ * @since_tizen 6.5
+ * @param[in] model The NNTrainer model handle to save
+ * @param[in] path_prefix  Path prefix to save the file. This function will save
+ * one or number of files using the given prefix. Note that this does not state
+ * the exact file name but prefix
+ * @param[in] option Option flag which part of model should be saved
+ * @return @c 0 on success, Otherwise a negative error value
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_INVALID_PARAMETER Path is a directory without a prefix, or
```

Comment:
write behavior if the file already exists - overwrite or error (overwrite is better IMO).

Reply:
adding a number would also increase another argument for load - load latest/ or a specific number.

Reply:
> write behavior if the file already exists - overwrite or error

if we overwrite, do we clean the previously saved ones?

for example
```cpp
 save("here", config | params)
 save("here", params) <--- here do we leave config? or delete config?
```

Reply:
we can leave it to the user - or we can use numbered saves
@jijoongmoon what do you recommend?

Reply:
Yeah, I guess we can leave this to the user. One vote for overwrite

Reply:
I guess numbered saves with limitations from uses.